digdag-standards/src/main/java/io/digdag/standards/operator/td/TdOperatorFactory.java

```diff
@@ -219,11 +219,13 @@ else if (createTable.isPresent()) {
                     throw new ConfigException("Unknown 'engine:' option (available options are: hive and presto): "+engine);
             }
 
+            final String sql = createStmtComment(request) + stmt;
```

Comment:
Why not return `StringBuilder` from the method to avoid this String concatenation?

Reply:
Good point, thank you. Updated: 9db104e